libraries/AP_HAL_ChibiOS/hwdef/CarbonixCommon/scripts/modules/bit_engine.lua

```diff
@@ -0,0 +1,57 @@
+local cx_msg = require("msg")
+local aircraft_type = require("aircraft")
+
+KELVIN_CELSIUS_DIFF = 273.15
+
+local Engine = {
+    name = "Engine",
+
+    -- These range is for pre-arm checks only. 
+    -- Post-arm checks are set in the Mission-Planner 
+    --   -> warning -> SETUP -> Advanced -> Warning Manager
+    efi_chtmin = 100,   -- in celcius
+    efi_chtmax = 150,   -- in celcius
+}
+
+-- Initialize the Engine module
+function Engine:init()
+    -- Volanti does not have Engine module
+    if aircraft_type == "Volanti" then
+        return
+    end
+    cx_msg:send(cx_msg.MAV_SEVERITY.INFO, "Engine init (" .. aircraft_type .. ")")
+end
+
+-- Update loop for Engine module
+function Engine:update()
+    -- Check is done in check_for_errors()
+    return
+end
```

Comment:
Luacheck is mad about this, because it has an unused input variable: `self`.

Maybe we could add a provision in build-in-test to allow modules to not define an update function at all if unused. Basically, when looping through the modules, nil-check their update member

Reply:
Temporarily changed this line : 
`cx_msg:send(cx_msg.MAV_SEVERITY.INFO, "Engine init (" .. aircraft_type .. ")")`
to this
`cx_msg:send(cx_msg.MAV_SEVERITY.INFO, self.name .. " init (" .. aircraft_type .. ")")`